src/main/java/amazin/controller/ApplicationController.java

```diff
@@ -15,27 +15,27 @@
 
     @GetMapping({ "/" })
     public String index(Model model) {
-        model.addAttribute("books", bookRepository.findAll());
+        model.addAttribute("book", bookRepository.findAll());
         return "index";
     }
 
-    @GetMapping({"/login"})
-    public String login(){
+    @GetMapping({ "/login" })
+    public String login() {
```

Comment:
not needed remove them

Reply:
this is done automatically by my linter, can't do anything about it

Reply:
When you go to commit you can add certain lines only from the changes to the commmit, this way you don't need to commit these next time I use a tool like git gui or gitg which I find makes it easy to stage only certain lines for a commit.

I agree with @Hajar1 maybe we should do an extra commit to revert those other line changes to this file for this PR